kubectl-plugin/pkg/cmd/job/job_submit.go

```diff
@@ -36,6 +36,7 @@ const (
 	dashboardAddr      = "http://localhost:8265"
 	clusterTimeout     = 120.0
 	portforwardtimeout = 60.0
+	fieldManager       = "ray-kubectl-plugin"
```

Comment:
let's put in https://github.com/ray-project/kuberay/blob/master/kubectl-plugin/pkg/util/constant.go and use everywhere else too?

Reply:
Done, looks like only one other mutating call in create_workergroup.go.